packages/paste-core/components/button-group/src/ButtonGroup.tsx

```diff
@@ -71,7 +71,7 @@ export interface ButtonGroupProps extends HTMLPasteProps<"div"> {
    * @memberof ButtonGroupProps
    */
   justifyContent?: JustifyOptions;
-  children: React.ReactElement[];
+  children: React.ReactElement<any>[];
```

Comment:
suggestion(non-blocking): ReactNode is a better type for children

```suggestion
  children: React.ReactNode;
```